README.md

```diff
@@ -8,30 +8,38 @@ This tool can currently be used to update the Black Magic Debug firmware on your
 
 ## Installation
 
+Binary releases for Linux, Mac (x64/ARM) and Windows (x64/ARM) are now available with every
```

Comment:
The proper name for the architectures are amd64 and AArch64, please use them in the ()'s (x64 is Microsoft's name for the architecture more formerly known as x86_64 or amd64, "ARM" refers normally to 32-bit ARM, aka AArch32)

Reply:
This was just me being lazy; fixed.